x-pack/plugins/security_solution/public/exceptions/components/link_to_list_details/index.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { FC } from 'react';
+
+import { SecuritySolutionLinkAnchor } from '../../../common/components/links';
+import { SecurityPageName } from '../../../../common/constants';
+
+interface LinkToListDetailsProps {
+  linkTitle: string;
+  listId: string;
+  external?: boolean;
+  dataTestSubj?: string;
+}
+// This component should be removed and moved to @kbn/securitysolution-exception-list-components
+// once all the building components get moved
+
+const LinkToListDetailsComponent: FC<LinkToListDetailsProps> = ({
+  linkTitle,
+  listId,
+  external,
+  dataTestSubj,
+}) => {
+  return (
+    <SecuritySolutionLinkAnchor
+      data-test-subj={`linkToRuleSecuritySolutionLink${dataTestSubj ?? ''}`}
+      external={external}
```

Comment:
I guess the `external` flag is for the _EuiLink_ to show the icon. If so, it is not needed when `target="_blank"` is present.
Also, `target="_blank"` is always set, maybe we should use a new `openInNewTab` prop